mesonbuild/interpreter/interpreter.py

```diff
@@ -2404,8 +2407,8 @@ def func_configure_file(self, node: mparser.BaseNode, args: T.List[TYPE_var],
                                               install_tag=install_tag, data_type='configure'))
         return mesonlib.File.from_built_file(self.subdir, output)
 
-    def extract_incdirs(self, kwargs):
-        prospectives = extract_as_list(kwargs, 'include_directories')
+    def extract_incdirs(self, kwargs, key = 'include_directories'):
```

Comment:
```suggestion
    def extract_incdirs(self, kwargs, key: str = 'include_directories'):
```
 
Let's just add the type annotation while we're here please :)